packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -305,6 +305,14 @@ class RenderingEngine implements IRenderingEngine {
 
     this.setVtkjsDrivenViewports(vtkDrivenViewportInputEntries);
     this.setCustomViewports(customRenderingViewportInputEntries);
+
+    // Making sure the setViewports api also can fill the canvas
```

Comment:
I suspect there are a couple of spots that the background colors are filled.  Any reason to not make a method and call it consistently for filling?

Reply:
There is already fillCanvasWithBackgroundColor I think this is good enough abstraction